app_flutter/test/status_grid_test.dart

```diff
@@ -96,48 +96,50 @@ void main() {
       expect(firstTask.task, taskMatrix.task(0, 0));
     });
 
+    /// Matrix Diagram:
+    /// ✓☐☐
+    /// ☐✓☐
+    /// ☐☐✓
+    final List<CommitStatus> statusesWithSkips = <CommitStatus>[
+      CommitStatus()
+        ..stages.add(Stage()
+          ..name = 'A'
+          ..tasks.addAll(<Task>[
+            Task()
+              ..name = '1'
+              ..status = TaskBox.statusSucceeded
+          ])),
+      CommitStatus()
+        ..stages.add(Stage()
+          ..name = 'A'
+          ..tasks.addAll(<Task>[
+            Task()
+              ..name = '2'
+              ..status = TaskBox.statusSucceeded
+          ])),
+      CommitStatus()
+        ..stages.add(Stage()
+          ..name = 'A'
+          ..tasks.addAll(<Task>[
+            Task()
+              ..name = '3'
+              ..status = TaskBox.statusSucceeded
+          ]))
+    ];
```

Comment:
I have difficulty in understanding how this commit status list be converted to the matrix diagram above.

Reply:
Would it be helpful if I added the comment below the diagram stating:
```
/// To construct the matrix from this diagram, each [CommitStatus] must have a unique [Task] that does not share its name with any other [Task].
```

Reply:
Yes, that would be helpful.

Reply:
Done.